data/boarding missions.txt

```diff
@@ -34,6 +371,163 @@ mission "Assisting Hai"
+	source
+		government "Hai (Unfettered)"
+	on offer
+		payment -300000
+		conversation
+			`After assisting the <origin> you find that 300000 credits have been removed from your account with a simple message saying, "Thank you for your contribution."`
+				decline
+	destination Darkcloak
+	
+mission "Assisting Unfettered Hai"
+	assisting
+	repeat
+	to offer
+		random < 8
+	source
+		government "Hai (Unfettered)"
+	on offer
+		payment 20000
+		conversation
+			`After assisting the <origin> you find that <payment> credits have been added to your account along with a message reading, "We accept true individuals prepared to support the unaltered Hai."`
+				decline
+	destination Darkcloak
+
+mission "Assisting Unfettered Hai (Strong)"
+	assisting
+	repeat
+	to offer
+		random < 8
+	source
+		government "Hai (Unfettered)"
+	on offer
+		payment 20000
+		conversation
+			`After assisting the <origin> you find that <payment> credits have been added to your account along with a message reading, "We welcome strong individuals such as yourself."`
+				decline
+	destination Darkcloak
+
+mission "Assisting Unfettered Hai (Peace)"
+	assisting
+	repeat
+	to offer
+		random < 8
+	source
+		government "Hai (Unfettered)"
+	on offer
+		payment 20000
+		conversation
+			`After assisting the <origin> you find that <payment> credits have been added to your account along with a message reading, "Peace is for the weak, we are glad you support that message."`
+				decline
+	destination Darkcloak
+	
+mission "Assisting Wanderers (Pre-Language)"
+	assisting
+	repeat
+	to offer
+		random < 15
+		not "language: Wanderer"
+	source
+		government Wanderer
+	on offer
+		payment 25000
+		conversation
+			`After assisting the <origin> you recieve <payment> credits and a message saying, "Keku iy yechu'ei iy veki'ai sat", which you interpret to be one of thanks.`
+				decline
+	destination "Vara K'chrai"
+
+mission "Assisting Wanderers (Language)"
+	assisting
+	repeat
+	to offer
+		random < 27
+		has "language: Wanderer"
+	source
+		government Wanderer
+	on offer
+		payment 89000
+		conversation
+			`Informed that you risked your livelihood to save the <origin>, the captain transfers you <payment> along with a message stating, "Thank you for [risking, sacrificing] yourself for us."`
+				decline
+	destination "Vara K'chrai"
+	
+mission "Assisting Efreti"
+	assisting
+	repeat
+	to offer
+		random < 7
+	source
+		government "Kor Efret"
+	on offer
+		payment 18000
+		conversation
+			`(An alien voice says something in a language you do not recognize.)`
+				decline
+	destination "Laki Nemparu"
+
+mission "Assisting Remnant"
+	assisting
+	repeat
+	to offer
+		random < 34
+	source
+		government Remnant
+	on offer
+		payment 43000
+		conversation
+			`In return for saving the <origin>, the captain gives you <payment>.`
+				decline
+	destination Aventine
+
+mission "Assisting Mereti"
+	assisting
+	repeat
+	to offer
+		random < 5
+	source
+		government "Kor Mereti"
+	on offer
+		payment 0
```

Comment:
If you get no payment then what's even the point in this one?

Reply:
Still this.

Reply:
Flavor, of course.